NewDawn/NewDawn.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What’s this?

Reply:
It seems like something we can ignore from source control, from what I googled.